lib/pbio/drv/ioport/ioport_lpf2.c

```diff
@@ -481,7 +481,20 @@ PROCESS_THREAD(pbdrv_ioport_lpf2_process, ev, data) {
     }
 
     while (true) {
-        PROCESS_WAIT_EVENT_UNTIL(ev == PROCESS_EVENT_TIMER && etimer_expired(&timer));
+        PROCESS_WAIT_EVENT_UNTIL(ev == PROCESS_EVENT_POLL || (ev == PROCESS_EVENT_TIMER && etimer_expired(&timer)));
```

Comment:
This can be simplified to `PROCESS_WAIT_EVENT()`

Then wrap the timer handler section in `if (ev == PROCESS_EVENT_TIMER && etimer_expired(&timer)) { }`

Reply:
:+1: